src/main/java/seedu/address/ui/ResultDisplay.java

```diff
@@ -36,6 +38,32 @@ public ResultDisplay() {
     private void handleNewResultAvailableEvent(NewResultAvailableEvent event) {
```

Comment:
Remember to include method header comments as method now has become more complicated.